src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Features/FeatureValues/create.html.twig

```diff
@@ -0,0 +1,49 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% set layoutTitle = 'Add a new feature value'|trans({}, 'Admin.Catalog.Feature') %}
```

Comment:
<img width="144" alt="Capture d’écran 2019-12-12 à 17 35 43" src="https://user-images.githubusercontent.com/32565890/70730758-dc858000-1d05-11ea-896e-7d9d4a939bd2.png">

Reply:
Are we talking about this CTA? If yes, the article should be removed to stay consistent with the other pages. ;-)